scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn_texas.yaml

```diff
@@ -159,6 +159,57 @@ Texas_S-PWQFN-N16_EP:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+Texas_S-PWQFN-N16_EP1.7x1.7:
+  device_type: 'QFN'
+  size_source: 'https://www.ti.com/lit/ds/symlink/tlv9064.pdf#page=44'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  custom_name_format: 'Texas_S-PWQFN-N{pincount}_EP{ep_size_x:g}x{ep_size_y:g}mm{vias:s}'
+  body_size_x:
+    minimum: 2.85
+    maximum: 3.15
+  body_size_y:
+    minimum: 2.85
+    maximum: 3.15
+  overall_height:
+    minimum: 0.7
+    maximum: 0.8
+
+  lead_width:
+    minimum: 0.18
+    maximum: 0.3
+  lead_len:
+    minimum: 0.3
+    maximum: 0.5
+
+  EP_size_x:
+    nominal: 1.7
+    tolerance: 0.1
+  EP_size_y:
+    nominal: 1.7
+    tolerance: 0.1
```

Comment:
Please use the package pad dimensions of 1.66mm for `EP_size_[xy]` and add `EP_size_[xy]_overwrite: 1.7` to get the suggested EP land size.